packages/rln/src/contract/constants.ts

```diff
@@ -25,4 +25,13 @@ export const RATE_LIMIT_PARAMS = {
   EPOCH_LENGTH: 600 // Epoch length in seconds (10 minutes)
 } as const;
 
+/**
+ * Default Q value for the RLN contract
+ * This is the upper bound for the ID commitment
+ * @see https://github.com/waku-org/specs/blob/master/standards/core/rln-contract.md#implementation-suggestions
+ */
+export const DEFAULT_Q = BigInt(
```

Comment:
It's not really "default", is it? It's just the Q value set.
```suggestion
export const RLN_Q = BigInt(
```